syntax = "proto3";

package face_aggregation;

import "common.proto";

service DataStorageService {
  rpc StoreFaceResult(FaceResultRequest) returns (DoneFlagToFaceAnalysisServiceResponse);
}